packages/react/src/components/Button/Button.tsx

```diff
@@ -2,10 +2,20 @@ import '@compassion-gds/css/src/components/Button/button.css';
 
 export interface ButtonProps {
   label: string;
+  mode?: 'primary' | 'secondary' | 'tertiary',
+  size?: 'small' | 'medium' | 'large',
+  disabled?: boolean,
+  onClick?: () => void,
```

Comment:
```suggestion
  onClick?: (event: MouseEvent<HTMLButtonElement>) => void,
```

Though you'd also have to add 

```js
import { MouseEvent } from 'react';
```